.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,24 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
+      - run: black --check . || true
+      - run: codespell  # --ignore-words-list="" --skip=""
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: isort --check-only --profile black . || true
+      - run: pip install tox -r test-requirements.txt
+      - run: pip install -e .
+      # - run: GS_CREDS_FILENAME=<YOUR_CREDS.json> make test
+      - run: make test || true
+      - run: nosetests -vv tests/test.py || true
```

Comment:
This should be removed too, now one can run the tests using `make test` which will run tox which will do the setup for each available version of python.

Reply:
Removed.